src/main/webapp/app/exercises/programming/hestia/git-diff-report/git-diff-file-panel.component.html

```diff
@@ -0,0 +1,33 @@
+<ngb-accordion activeIds="custom-panel-1" [destroyOnHide]="false" class="git-diff-file-panel">
+    <ngb-panel id="custom-panel-1" class="similar-post-list">
+        <ng-template ngbPanelHeader let-opened="opened">
+            <div class="d-flex align-items-center justify-content-between">
+                <div class="my-2 ps-1">
+                    <div *ngIf="filePath && previousFilePath && filePath !== previousFilePath; else oneFileName">{{ previousFilePath }} → {{ filePath }}</div>
```

Comment:
Should we somehow use an escape character for `→`? Not sure right now if this might be a source of error.

Reply:
It should be fine as it is, as the arrow is not anything used in the HTML syntax.